src/common/drpc.c

```diff
@@ -188,18 +191,20 @@ unixcomm_listen(char *sockaddr, int flags)
 	struct unixcomm		*comm;
 
 	comm = new_unixcomm_socket(flags);
-	if (comm == NULL) {
+	if (comm == NULL)
 		return NULL;
-	}
 
 	fill_socket_address(sockaddr, &address);
 	if (bind(comm->fd, (struct sockaddr *)&address,
-			sizeof(struct sockaddr_un)) < 0) {
+		 sizeof(struct sockaddr_un)) < 0) {
+		D_ERROR("Failed to bind socket fd %d\n", comm->fd);
```

Comment:
I agree with @liw that it would be nice to capture errno when possible. Debugging this failure would be a lot easier for the admin if they saw that it didn't work because e.g. the socket dir doesn't exist.

Reply:
Went ahead and logged the errno in places where it's relevant.